### PR TITLE
Update arch.json - Adapt owl ID

### PR DESCRIPTION
**Pull request by Arch881010**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

(NOTE: Currently in the template for a PR, the note for "sufficent information" states you *"need to have your email presented at `email` field."

### CODE REVIEW

**Comment by Arch881010** [comment]
Pending re-work to follow the probably new owner field. #14802 for reference. (Left at draft if owl ID gets implemented.)

File: domains/_github-pages-challenge-Arch881010.arch.json [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
     "username": "Arch881010",
-    "email": "c1yd4.arch@inbox.testmail.app",
+    "email": "arch@arch1010.dev",
     "discord": "arch1010",
-    "discord-id": "410248634593050627"
+    "discord-id": "410248634593050627",
+    "note": "OLD Owner details, leaving email the same to further find phishing domain endings. (!! See arch.json)"
   },
   "record": {
+    "CNAME": "arch881010.github.io",
```
Comment (wdhdev) [inline domains/_github-pages-challenge-Arch881010.arch.json:14]: CNAME + TXT is not possible.

File: domains/_github-pages-challenge-Arch881010.arch.json [hunk 2]
```diff
@@ -1,13 +1,17 @@
 {
   "description": "My personal website on arch.is-a.dev (main files in repo/repository)[Github Challenge].",
   "repo": "https://github.com/Arch881010/Arch881010",
-  "owner": {
+  "owner": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7InVzZXJfaWQiOiIxMTE3ODkzMTAiLCJ1c2VybmFtZSI6IkFyY2g4ODEwMTAiLCJlbWFpbCI6ImFyY2gxMDEwQGljbG91ZC5jb20ifSwiZXhwaXJlc0luIjoiOTk5OXkiLCJpYXQiOjE3MjE4NzIxNzB9.UIfmbJXRHNXMi1l3P5hYx6d4IdunEjxUiXDU_XwZHJc",
+  "note": "This is the ID from owl (https://owl.is-a.dev/id/111789310/Arch881010)",
```
Comment (wdhdev) [inline domains/_github-pages-challenge-Arch881010.arch.json:5]: Owl ID is not complete and is going to be changing to a different encryption method soon.

File: domains/arch.json [hunk 3]
```diff
@@ -1,11 +1,14 @@
 {
   "description": "My personal website on arch.is-a.dev (main files in repo/repository).",
   "repo": "https://github.com/Arch881010/Arch881010",
-  "owner": {
+  "owner": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7InVzZXJfaWQiOiIxMTE3ODkzMTAiLCJ1c2VybmFtZSI6IkFyY2g4ODEwMTAiLCJlbWFpbCI6ImFyY2gxMDEwQGljbG91ZC5jb20ifSwiZXhwaXJlc0luIjoiOTk5OXkiLCJpYXQiOjE3MjE4NzIxNzB9.UIfmbJXRHNXMi1l3P5hYx6d4IdunEjxUiXDU_XwZHJc",
+  "note": "This is the ID from owl (https://owl.is-a.dev/id/111789310/Arch881010)",
```
Comment (wdhdev) [inline domains/arch.json:5]: Owl ID is not complete and is going to be changing to a different encryption method soon.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by Arch881010** [comment]
Since encryption change, closing.

**Pull request closed without merging**